mypy/build.py

```diff
@@ -1794,8 +1794,13 @@ def patch_dependency_parents(self) -> None:
         semantic analyzer will perform this patch for us when processing stale
         SCCs.
         """
+        Analyzer = Union[SemanticAnalyzerPass2, NewSemanticAnalyzer]  # noqa
```

Comment:
What's this change? Is is related to the rest of this PR?

Reply:
This is needed because otherwise incremental tests crash with the new analyzer (I added couple incremental tests) with something like `'BuildManager' doesn't have attribute 'semantic_analyzer'`.